data2sqlite.py

```diff
@@ -45,8 +45,9 @@
 
 import_parser=subparsers.add_parser('file', help="Add to Source from other files")
 import_parser.add_argument('--sqlite', nargs='*', default=[],                   help='Filename for SQLite Database')
-import_parser.add_argument('--XML', nargs='*', default=[],                      help='Filename for XML')
-import_parser.add_argument('--CSV', nargs='*', default=[],                      help='Filename for CSV')
+import_parser.add_argument('--XML', nargs='*', default=[],                      help='Filename(s) for XML')
+import_parser.add_argument('--CSV', nargs='*', default=[],                      help='Filename(s) for CSV')
+import_parser.add_argument('--excel', nargs='*', default=[],                    help='Filename(s) for Excel')
```

Comment:
the help can list the supported formats

Reply:
- I added the supported formats as they are supported by the library
- `ods` is not supported